src/CreaTeBME/SensorManager.py

```diff
@@ -46,6 +46,8 @@ def start(self) -> None:
             self._thread.daemon = True
             self._thread.start()
         self._is_running = True
+        while len(self.get_measurements()[list(self._queue.keys())[0]]) == 0:
```

Comment:
Add to the doc of the start function that it is blocking until all sensors are sending data.